models/gc_tiger_lb2/config/default.yml

```diff
@@ -0,0 +1,31 @@
+general:
+  data_base_dir: /app/data
+  version: 1.0
+  description: Tiger challenge winner LB2 (dicom:sm to json with TIL score)
+
+execute:
+- DicomImporter
+- TiffConverter
+- TigerLB2Runner
+- ReportExporter
+- DataOrganizer
+
+modules:
+  DicomImporter:
+    source_dir: input_data
+    import_dir: sorted_data
+    sort_data: True
+    meta: 
+      mod: sm
```

Comment:
`mod: "%Modality"` can be used to automatically parse the dicom Modality header.

Reply:
I rather keep it at `sm`, since that should be the correct DICOM modality anyway for this pipeline to run as all inputs should be slide microscopy images. Is there any benefit to changing it here, except that nothing runs if it does not match the modality?

Reply:
Yes, e.g. Patient folder contains two dicom series, one SM and the other CT. It makes no sense to import both as SM. 

Please change as suggested.

Reply:
Ok, I'll update it as suggested.